services/build.gradle

```diff
@@ -42,13 +42,13 @@ dependencyManagement {
     }
 }
 
-jacocoTestReport {
-    reports {
-        xml.enabled = true
-        html.enabled = true
-        html.destination file("${buildDir}/jacocoHtml")
-    }
-}
+//jacocoTestReport {
```

Comment:
It is jacoco taks causing trouble? If so please delete the code instead of comment code. We can use GitHub commits history in case we want to bring it back.

Reply:
Yes, we'll remove it